raiden/network/proxies/token_network.py

```diff
@@ -114,7 +114,7 @@ def new_netting_channel(
             settle_timeout: The settle timout to use for this channel.
 
         Returns:
-            The address of the new netting channel.
+            The ChannelID (int) of the new netting channel.
```

Comment:
stating the type of the channel id in the comment is not needed. It's redundant information. (if god forbid we ever needed to change it again that comment would be wrong)